src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -31,23 +31,24 @@
     public static Candidate[] getSamplePersons() {
         return new Candidate[] {
             new Candidate(new StudentId("E0123456"), new Name("Alex Yeoh"), new Phone("87438807"),
-                new Email("E0123456@u.nus.edu"), new Course("Business Analytics"), new Seniority(1),
-                    getTagSet("friends"), PENDING, INTERVIEW_NOT_SCHEDULED, new Availability("1,2,3,4,5,6,7")),
+                    new Email("E0123456@u.nus.edu"), new Course("Business Analytics"), new Seniority(1),
+                    getTagSet("friends"), PENDING, INTERVIEW_NOT_SCHEDULED, new Availability("1,2,3,4,5")),
             new Candidate(new StudentId("E0234567"), new Name("Bernice Yu"), new Phone("99272758"),
-                new Email("E0234567@u.nus.edu"), new Course("Computer Engineering"), new Seniority(2),
-                getTagSet("colleagues", "friends"), PENDING, INTERVIEW_NOT_SCHEDULED, new Availability("1,2,3,4,5")),
+                    new Email("E0234567@u.nus.edu"), new Course("Computer Engineering"), new Seniority(2),
+                    getTagSet("colleagues", "friends"), PENDING, INTERVIEW_NOT_SCHEDULED,
+                    new Availability("1,2,3,4,5")),
             new Candidate(new StudentId("E0345678"), new Name("Charlotte Oliveiro"), new Phone("93210283"),
-                new Email("E0345678@u.nus.edu"), new Course("Computer Science"), new Seniority(3),
+                    new Email("E0345678@u.nus.edu"), new Course("Computer Science"), new Seniority(3),
                     getTagSet("neighbours"), PENDING, INTERVIEW_NOT_SCHEDULED, new Availability("1,2,3")),
             new Candidate(new StudentId("E0456789"), new Name("David Li"), new Phone("91031282"),
-                new Email("E0456789@u.nus.edu"), new Course("Information Security"), new Seniority(4),
+                    new Email("E0456789@u.nus.edu"), new Course("Information Security"), new Seniority(4),
                     getTagSet("family"), PENDING, INTERVIEW_NOT_SCHEDULED, new Availability("3,4,5")),
             new Candidate(new StudentId("E0567890"), new Name("Irfan Ibrahim"), new Phone("92492021"),
-                new Email("E0567890@u.nus.edu"), new Course("Information Systems"), new Seniority(1),
-                    getTagSet("classmates"), PENDING, INTERVIEW_NOT_SCHEDULED, new Availability("4,5,6")),
```

Comment:
Thanks for re-looking through the sample data and updating the availabilities for the candidates!!

Reply:
Just trying to do my part :)